tests/functional/aws-node-sdk/test/multipleBackend/delete/deleteAwsVersioning.js

```diff
@@ -102,14 +102,16 @@ function _deleteDeleteMarkers(s3, bucket, key, deleteMarkerVids, cb) {
 }
 
 function _awsGetLatestVerId(key, body, cb) {
-    return awsS3.getObject({ Bucket: awsBucket, Key: key }, (err, result) => {
-        assert.strictEqual(err, null, 'Expected success ' +
-            `getting object from AWS, got error ${err}`);
-        const resultMD5 = expectedETag(result.Body, false);
-        const expectedMD5 = expectedETag(body, false);
-        assert.strictEqual(resultMD5, expectedMD5);
-        return cb(null, result.VersionId);
-    });
+    const getObject = awsS3.getObject.bind(awsS3);
+    return setTimeout(getObject, 10000, { Bucket: awsBucket, Key: key },
```

Comment:
not super big, but I think we were trying to pull time limits out into variables if they're used more than once, so we only need to change them in one spot if the flakiness gets worse (or better!)

Reply:
I prefer to merge for now because we have so many prs lined up for merging; addressing the flakiness will help the other prs get merged, while we can always do a clean-up pr to pull the time limits into a variable (I have a clean-up pr planned that I could stick the commit into without too much trouble)